src/packages/@ncigdc/modern_components/BiospecimenCard/reducers.js

```diff
@@ -0,0 +1,52 @@
+export default [
```

Comment:
can we move this reducer to the `/dux` folder with the other reducers?

Reply:
It's a reducer specific to this component alone, i.e. not reusable/global.
By Component Oriented Programming principles, it should remain where it is, yet I'll move it if everyone else agrees I should.

Reply:
i'd prefer sticking with code organization we currently have in the portal (with all the reducers in one folder), if we choose as a team to reorganize at some point we can change things up then.